### PR TITLE
[HOTFIX] Pixel error fix

### PR DESCRIPTION
**Pull request by AtherBilal**

### CODE REVIEW

**Review by gabe647 — APPROVED** [review]
💯

**Comment by stazrad** [comment]
@AtherBilal can you bump `version` in `package.json` & `package-lock.json` to `1.17.1` (may need to update a snapshot as well)

**Review by stazrad — CHANGES_REQUESTED** [review]
bump version -- i made a release doc

File: src/Popup/utils.js [hunk 1]
```diff
@@ -138,7 +141,10 @@ const wmsSelector = (layer, event) => {
     if (features.length) {
       resolve({ features, layer })
     } else {
-      resolve({ features: [getPixelValue(layer, event)], layer })
+      const pixelValue = getPixelValue(layer, event)
+      const pixelFeatures = !!pixelValue ? [pixelValue] : []
```
Comment (IDisposable) [inline src/Popup/utils.js:145]: Won't this get wonky if the `pixelValue === 0` (e.g. falsey)

Comment (stazrad) [reply src/Popup/utils.js:145]: @IDisposable yea totally considered this- meant to check on the return type from `getPixelValue` ...lack of typescript strikes again 😩

**Review by akuma1 — APPROVED** [review]

**Comment by gabe647** [comment]
Per code review we will leave the `getPixelValue` function exported but remove the call sites in ol-kit.  This will keep the default click behavior consistent.

**Comment by gabe647** [comment]
As per discussion in planning meeting, we need to address the issue where clicking on an empty area briefly displays the popup with loading text.

**Comment by gabe647** [comment]
Reverted the popup behavior to not show when user clicks on an empty area of the map.

**Review by gabe647 — APPROVED** [review]
This works locally with geokit

**Review by akuma1 — APPROVED** [review]

**Review by stazrad — APPROVED** [review]

**Pull request merged**